podium-gateway/src/main/webapp/app/admin/health/modal/health-modal.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+
+import { HealthKey, HealthDetails } from '../health.model';
+
+@Component({
+  selector: 'pdm-health-modal',
+  templateUrl: './health-modal.component.html',
+})
+export class HealthModalComponent {
+  health?: { key: HealthKey; value: HealthDetails };
+
+  constructor(public activeModal: NgbActiveModal) {}
+
+  readableValue(value: any): string {
+    if (this.health?.key === 'diskSpace') {
+      // Should display storage space in an human readable unit
+      const val = value / 1073741824;
+      if (val > 1) {
+        return val.toFixed(2) + ' GB';
+      } else {
+        return (value / 1048576).toFixed(2) + ' MB';
```

Comment:
Hm, interesting that not ISO standards are followed here. This is from jhipster, right?
Correct would be:
```suggestion
        return (value / 1000000).toFixed(2) + ' MB';
```

Reply:
Right, these admin components are auto-generated and have little (if any) changes from my side.